simplemma/langdetect.py

```diff
@@ -4,23 +4,14 @@
 
 from collections import Counter
 from operator import itemgetter
-from typing import List, Optional, Tuple, Union
-
-from .simplemma import LANG_DATA, _control_lang, _load_data, _return_lemma
+from typing import List, Optional, Tuple
 
+from .lemmatizer import _return_lemma
+from .dictionaries import update_lang_data
 
 SPLIT_INPUT = re.compile(r"[^\W\d_]{3,}")
 
 
-def _update_lang_data(lang: Optional[Union[str, Tuple[str]]]) -> None:
-    # convert string
-    lang = _control_lang(lang)
-    # load corresponding data
-    global LANG_DATA
-    if not LANG_DATA or tuple(l.code for l in LANG_DATA) != lang:
-        LANG_DATA = _load_data(lang)
-
-
 def prepare_text(text: str) -> List[str]:
```

Comment:
What if we want to offer the users a possibility to customize the sampling logic?
How could we do this without globally overriding this method?

Reply:
I'm not sure what that means, but we can disuss it further in #33 if necessary.

Reply:
You're right, I just opened a corresponding issue.